_layouts/main.html

```diff
@@ -4,6 +4,7 @@
     {% include meta.html %}
     {% include styles.html %}
     {% include google_analytics.html %}
+    {% include google_recaptcha.html %}
```

Comment:
Would it be worth conditionally including this in only the contact page? To minimize extra JS loads, etc etc?

Reply:
@zachmargolis Lets get this merged/deployed and I will follow up with another PR

Reply:
Looks like we can use jekyll front-matter for this, however, I think we should test this out in a follow up PR.